SwiftLeeds/Views/About/AboutView.swift

```diff
@@ -9,7 +9,15 @@ import SwiftUI
 
 struct AboutView: View {
     var body: some View {
-        Text("About")
+        SwiftLeedsContainer {
+            ScrollView {
+                content
+            }
+        }
+    }
+
+    var content: some View {
```

Comment:
Should this be private?

Reply:
Certainly could be! Should is debated a bunch - happy to go with the consistency.

Reply:
It was a polite comment but if you'd prefer I'll go with `This should be private as not used outside the struct` if clarity is preferred.

Reply:
The interesting debate is around splitting them up like this, vs using private structs, vs just having them public because it doesnt really matter in a struct

I quite like it this way, which seems to _just about_ match how you did it